content/modules/mobile-travel-tutorial/partials/installation/manual.adoc

```diff
@@ -116,17 +149,16 @@ image::fts.png[]
 
 == Sync Gateway
 
+In this section, you will install and launch version 3.0.3 of Sync Gateway.
 
-In this section, you will install and launch version 3.0.0 of Sync Gateway.
-
-* Download Sync Gateway 3.0.0 from
+* Download Sync Gateway 3.0.3 from
 https://www.couchbase.com/downloads[here]
 for your platform
 
-* The Sync Gateway will have to be launched with the config file named `sync-gateway-config-travelsample.json` that you should have downloaded as per the instructions in the <<Workshop Repo>> section.
+* The Sync Gateway will have to be launched with the config file named `sync-gateway-config-travelsample-manual.json` that you should have downloaded as per the instructions in the <<Workshop Repo>> section.
 The config file will be located in `/path/to/mobile-travel-sample`.
 
-* Open the `sync-gateway-config-travelsample.json` and confirm that the RBAC user credentials configured on the Couchbase Server are used by Sync Gateway for accessing the bucket
+* Open the `sync-gateway-config-travelsample-manual.json` and confirm that the RBAC user credentials configured on the Couchbase Server are used by Sync Gateway for accessing the bucket
```

Comment:
Is the difference between the `sync-gateway-config-travelsample.json` and `sync-gateway-config-travelsample-manual.json` file the couchbase server address?

Reply:
yes `"server": "couchbases://localhost"`. 

`sync-gateway-config-travelsample.json` and `sync-gateway-config-travelsample 2.json` are duplicated for some reason.... I can remove `sync-gateway-config-travelsample 2.json` as is not used anywhere and I don't understand why it was pushed in the first place -> commit `362b4746c84d6fa9502020d56145b823e739e17a`

Reply:
I will transform it in `sync-gateway-config-travelsample-docker.json` and will be part of the PR related to docker.